src/VisualStudio/Core/Def/Implementation/Venus/ContainedLanguage.cs

```diff
@@ -85,6 +85,7 @@ internal partial class ContainedLanguage
                    projectId,
                    project,
                    GetFilePathFromHierarchyAndItemId(hierarchy, itemid),
+                   InternalLanguageNames.TypeScript,
```

Comment:
Not project.Language in case for some reason they do some change?

Reply:
project may be null.